apps/s_server.c

```diff
@@ -2030,9 +2029,9 @@ static int sv_body(int s, int stype, unsigned char *context)
     unsigned long l;
     SSL *con = NULL;
     BIO *sbio;
-    struct timeval timeout;
+    struct timeval timeout = { 0 };
 #if defined(OPENSSL_SYS_WINDOWS) || defined(OPENSSL_SYS_MSDOS)
-    struct timeval tv;
+    struct timeval tv = { 0 };
```

Comment:
These are locals and do not have a default initialization ... but is it better to memset() to zero instead of using an incomplete initializer?